docs/reference/monitoring/configuring-filebeat.asciidoc

```diff
@@ -117,7 +117,7 @@ If {security-features} are enabled, you must provide a valid user ID and
 password so that {filebeat} can connect to {kib}: 
 
 .. Create a user on the monitoring cluster that has the 
-{stack-ov}/built-in-roles.html[`kibana_user` built-in role] or equivalent
+{stack-ov}/built-in-roles.html[`kibana_admin` built-in role] or equivalent
```

Comment:
Does filebeat need to connect as `kibana_admin` or would a less privileged role suffice ?

Reply:
@legrego What's our recommendation here? What privileges does filebeat need in order to use the Kibana API to setup dashboards etc?